### PR TITLE
Refactored main.js

### PR DESCRIPTION
**Pull request by shichongrui**

### CODE REVIEW

File: src/dashboard/index.js [hunk 1]
```diff
@@ -6,10 +6,9 @@ import DashboardMain from './DashboardMain'
 const { remote } = require('electron')
 const app = remote.app
 
-import i18n from 'format-message'
-i18n.setup({
-  translations: require('../../locales'),
-  locale: app.getLocale() || 'en'
+i18n.setup({
```
Comment (cameronsutter) [inline src/dashboard/index.js:9]: it looks like this file doesn't import i18n any more

File: bin/main_modules/menus/plottr.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      }
+    },
+    {
+      label: 'View Process Manager',
+      visible: NODE_ENV === 'dev',
+      click: () => openProcessManager()
+    },
+    {
+      label: i18n('Language'),
+      click: () => {
```
Comment (cameronsutter) [inline bin/main_modules/menus/plottr.js:96]: is this where they are going to choose their language? I'm guessing this PR is just so I can see the refactor, but it's not ready to merge, right?

**Review by cameronsutter — COMMENTED** [review]
Really liking how you organized things. I wasn't sure if moving some of those things to their own modules would be possible. Well done

File: bin/main_modules/menus/plottr.js (hunk 2 above)
Comment (shichongrui) [reply bin/main_modules/menus/plottr.js:96]: I'm going to do language selection in a different PR. I didn't mean for this to be included here, so I will remove it for this PR so this one can focus on the refactor.

File: src/dashboard/index.js (hunk 1 above)
Comment (shichongrui) [reply src/dashboard/index.js:9]: I'll fix it.

**Pull request merged**